clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy

```diff
@@ -220,6 +219,16 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
       .build()
   }
 
+  long getCreationTime(Stack stack) {
```

Comment:
I thought about after I deleted DateUtils and found that the logic left in place got a little uglier.